circle.yml

```diff
@@ -0,0 +1,72 @@
+---
+version: 2
+
+buildSteps: &buildSteps
+  - checkout
+  - run:
+      name: install-npm
+      command: npm install
+  - run:
+      name: test
+      command: npm test
```

Comment:
From [package.json](https://github.com/resin-io-modules/docker-qemu-transpose/blob/master/package.json#L8-L10) it doesn't look like this will actually build. Probably want to do both here (or change package.json to build automatically)